CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Foreman Documentation
+
+The Foreman Community welcomes all feedback, issues, pull requests (PRs), and reviews.
+Every contributor has different backgrounds, interests, and experiences with Foreman and its open source community.
+
+We give ourselves these guidelines to collaborate and contribute to Foreman documentation more efficiently.
+It helps everyone to set expectations, communicate conventions, and ensures happy collaborative work across organisations and backgrounds.
+We all respect each others time and energy spent on Foreman documentation.
+
+## Maintainers
+
+* help less experienced community members with git, Github, PRs, asciidoc, and asciidoctor.
+* expect a basic effort in contributions.
+* only merge PRs if the Github Action is green.
+* read the proposed change and make friendly and helpful comments.
+* ping community members with more experience if they are unsure about specific details.
+* try to review PRs in a timely manner.
+
+## Contributors
+
+* are able to make meaningful contributions to the Foreman documentation, regardless of their experience with git, asciidoc, writing documentation, or Foreman.
+* familiarise themselves with the [Pull Request Checklist](#Pull-Request-Checklist).
+* open additional issues if a contribution is incomplete.
+* are invited to review other PRs.
+
+## Pull Request Checklist
+
+* [ ] I am familiar with the conventions as specified in the [guides/README.md](guides/README.md) file.
+* [ ] Before pushing, I view my diff to master or the target branch and check for spelling mistakes, broken conflict resolutions, etc.
```

Comment:
[ ] Before pushing, I view my diff **against** master or the target branch and check for spelling mistakes, **conflicting merges**, etc.